packages/@uppy/core/types/index.d.ts

```diff
@@ -262,24 +268,27 @@ export type RestrictionFailedCallback<TMeta extends IndexedObject<any>> = (
 export interface UppyEventMap<
   TMeta extends IndexedObject<any> = Record<string, unknown>,
 > {
+  'cancel-all': GenericEventCallback
```

Comment:
Generic, because we don't have a special type for it like with others?

Reply:
A few of them are generic, I'm not sure what the reason is, it was done as part of https://github.com/transloadit/uppy/pull/3085

Reply:
It's for all events that are not unique and have the same type. It's just for reusability.